build-logic/performance-testing/src/main/kotlin/gradlebuild/performance/tasks/BuildCommitDistribution.kt

```diff
@@ -71,27 +69,33 @@ abstract class BuildCommitDistribution @Inject internal constructor(
     @get:Optional
     abstract val commitBaseline: Property<String>
 
-    @get:OutputDirectory
-    abstract val commitDistributionHome: DirectoryProperty
+    @get:OutputFile
+    abstract val commitDistribution: RegularFileProperty
 
     @get:OutputFile
     abstract val commitDistributionToolingApiJar: RegularFileProperty
 
     init {
         onlyIf { commitBaseline.getOrElse("").matches(commitVersionRegex) }
-        commitDistributionHome.set(project.layout.buildDirectory.dir(commitBaseline.map { "distributions/gradle-$it" }))
-        commitDistributionToolingApiJar.set(project.layout.buildDirectory.file(commitBaseline.map { "distributions/gradle-tooling-api-$it.jar" }))
+        commitDistribution.set(project.rootProject.layout.projectDirectory.file(commitBaseline.map { "intTestHomeDir/commit-distributions/gradle-$it.zip" }))
+        commitDistributionToolingApiJar.set(project.rootProject.layout.projectDirectory.file(commitBaseline.map { "intTestHomeDir/commit-distributions/gradle-tooling-api-$it.jar" }))
```

Comment:
I think this configuration should go the plugin and shouldn't happen in the constructor of the task.